app/pb/communities.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package org.couchers.api.communities;
+
+import "google/protobuf/timestamp.proto";
+
+import "pb/api.proto";
+
+message Point {
+  double lat = 1;
+  double lng = 2;
+}
+
+message Node {
+  int32 node_id = 1;
```

Comment:
Other proto files use `uint64` for ids (and `uint32` for counts, by the way). This is poorly documented, sorry!

Reply:
What should be used then? In models.py is this id described as Integer which translates to PostgreSQL as an int32. So it is best to use that here too. Why using uint32 or uint64 instead?

Reply:
PostgreSQL does not have unsigned integers. And in models.py is Integer used. That's why I think int32 is best.

Reply:
The largest dataset at https://simplemaps.com/data/world-cities contains 4.2 million places, so 31 bits should be enough to identify a unique node.

Reply:
We should stick to consistency and not pick id sizes based on the max number of entries. A few bytes here or there don't matter in this context.

Reply:
Does this mean you want the datatype in models.py to be changed to `BigInteger`?

Reply:
If someone would like to do the refactoring job of replacing all the integer `id`s in the models with big serials, that's OK. But it doesn't matter if it's the same as elsewhere and uses `Integer`.

Reply:
Ok, so I understand you would like to use uint64 here, which I find strange as the model and db uses int32 and in js it is translated as float64 which does not cover uint64. To keep it consistently wrong is not a big enough reason for me to not do it right from now on. The future refactoring job will grow and grow.

Reply:
Then please refactor all `id`s in `models.py` to be big serials and we'll start doing it right everywhere. That's better as a separate PR off `develop`, I'd say